core/unit_test/TestGraph.hpp

```diff
@@ -174,6 +174,31 @@ TEST_F(TEST_CATEGORY_FIXTURE(graph), can_instantiate_only_once) {
   }
 }
 
+// This test submits on an execution space instance different from the
+// one passed to the Kokkos::Graph constructor.
+TEST_F(TEST_CATEGORY_FIXTURE(graph),
+       submit_onto_another_execution_space_instance) {
```

Comment:
Potentially skip it if the exec space is HIP and the ROCM version is 5.2

Reply:
Under `ROCm 5.2`, `Kokkos` uses the default graph implementation instead of the specialization for the `HIP` backend:https://github.com/kokkos/kokkos/blob/ed20bf11df3bedecff46d2bef12f5a97f486a769/core/src/Kokkos_Graph.hpp#L170-L173

Therefore, the test makes no sense and can be skipped.

@Rombur @dalg24